src/main/java/com/github/jreddit/oauth/param/RedditDuration.java

```diff
@@ -0,0 +1,27 @@
+package com.github.jreddit.oauth.param;
+
+/**
+ * Enumerator for the duration of tokens.<br>
```

Comment:
I personally find html tags in javadoc hard to read if you are inside an IDE/text editor while developing/reading library doc.